repository/mock/release.go

```diff
@@ -25,3 +25,8 @@ func (m *ReleaseRepository) ReadForProject(ctx context.Context, projectID, relea
 	}
 	return args.Get(0).(svcmodel.Release), args.Error(1)
 }
+
+func (m *ReleaseRepository) Delete(ctx context.Context, projectID, releaseID uuid.UUID) error {
```

Comment:
Naming will be updated once the conclusion is reached here - https://github.com/jan-zabloudil/release-manager/pull/69#pullrequestreview-2059661739